frontend/src/pages/Contracts/index.tsx

```diff
@@ -1,14 +1,13 @@
 import React from "react";
```

Comment:
Not sure how this is related to adding a new button.. 
try not to add multiple things to the same pr, especially if they might not be related, or maybe they are.
Or at least give the reviewer the reason you are updating this files.